crates/roc_std/src/roc_str.rs

```diff
@@ -325,7 +325,9 @@ impl RocStr {
                 }
             }
             RocStrInnerRef::SmallString(small_str) => {
-                let mut bytes = small_str.bytes;
+                let mut bytes = [0; size_of::<RocList<u8>>()];
+                let mut it = small_str.bytes.iter();
+                bytes = bytes.map(|_| it.next().copied().unwrap_or_default());
 
```

Comment:
this code in effect borrowed too few bytes (only 23, not the 24 that are needed)

Reply:
Off-by-one errors! :rage1: